samples/acl.js

```diff
@@ -267,27 +258,26 @@ function printFileAclForUser (bucketName, filename, userEmail) {
 
   const options = {
     // Specify the user
-    entity: `user-${userEmail}`
+    entity: `user-${userEmail}`,
```

Comment:
What's the deal with these trailing commas?

Reply:
I set prettify to `trailingComma: es5`. That is an item that is configurable in prettify.

Reply:
What I meant is, can we undo the trailing commas?

Reply:
Hmm, I do not consider it binding for open source, but:
https://google.github.io/styleguide/jsguide.html#features-arrays-trailing-comma
https://google.github.io/styleguide/jsguide.html#features-objects-use-trailing-comma

Reply:
alright